services/user/user.go

```diff
@@ -58,7 +58,7 @@ func RenameUser(ctx context.Context, u *user_model.User, newUserName string) err
 			u.Name = oldUserName
 			return err
 		}
-		return nil
+		return repo_model.UpdateRepositoryOwnerName(ctx, oldUserName, newUserName)
```

Comment:
I guess it could be `repo_model.UpdateRepositoryOwnerNames(u.ID, newUserName)` ?

Reply:
Hmm, I think the `UpdateRepositoryOwnerName` could be removed and totally replaced by `UpdateRepositoryOwnerNames`

Reply:
Done